libraries/entities/src/EntitiesAuditLogging.cpp

```diff
@@ -0,0 +1,84 @@
+//
+//  EntitiesAuditLogging.cpp
+//  libraries/entities/src
+//
+//  Created by Kalila L on Feb 5 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#include "EntitiesAuditLogging.h"
+
+#include <QJsonObject>
+#include <QTimer> 
+
+Q_LOGGING_CATEGORY(entities_audit, "vircadia.entities.audit");
+
+QJsonObject auditLogAddBuffer;
+QJsonObject auditLogEditBuffer;
```

Comment:
are QJsonObjects the most efficient way to store this data?  it seems like an std::unordered_map or something would be simpler